advocacy_docs/partner_docs/HashicorpVault/05-UsingHashicorpVault.mdx

```diff
@@ -0,0 +1,241 @@
+The files belonging to this database system will be owned by user "enterprisedb".
+This user must also own the server process.
+The database cluster will be initialized with locale "C.UTF-8".
+The default database encoding has accordingly been set to "UTF8".
+The default text search configuration will be set to "english".
+Data page checksums are disabled.
+Transparent data encryption is enabled.
+creating directory /var/lib/edb-as/15/dd12 ... ok
+creating subdirectories ... ok
+selecting dynamic shared memory implementation ... posix
+selecting default max_connections ... 100
+selecting default shared_buffers ... 128MB
+selecting default time zone ... America/New_York
+creating configuration files ... ok
+setting up data encryption ... ok
+running bootstrap script ... ok
+performing post-bootstrap initialization ... ok
+creating edb sys ... ok
+loading edb contrib modules ...
+edb_redwood_bytea.sql 
+edb_redwood_date.sql 
+dbms_alert_public.sql 
+dbms_alert.plb 
+dbms_job_public.sql 
+dbms_job.plb 
+dbms_lob_public.sql 
+dbms_lob.plb 
+dbms_output_public.sql 
+dbms_output.plb 
+dbms_pipe_public.sql 
+dbms_pipe.plb 
+dbms_rls_public.sql 
+dbms_rls.plb 
+dbms_sql_public.sql 
+dbms_sql.plb 
+dbms_utility_public.sql 
+dbms_utility.plb 
+dbms_aqadm_public.sql 
+dbms_aqadm.plb 
+dbms_aq_public.sql 
+dbms_aq.plb 
+dbms_profiler_public.sql 
+dbms_profiler.plb 
+dbms_random_public.sql 
+dbms_random.plb 
+dbms_redact_public.sql 
+dbms_redact.plb 
+dbms_lock_public.sql 
+dbms_lock.plb 
+dbms_scheduler_public.sql 
+dbms_scheduler.plb 
+dbms_crypto_public.sql 
+dbms_crypto.plb 
+dbms_mview_public.sql 
+dbms_mview.plb 
+dbms_session_public.sql 
+dbms_session.plb 
+edb_bulkload.sql 
+edb_gen.sql 
+edb_objects.sql 
+edb_redwood_casts.sql 
+edb_redwood_strings.sql 
+edb_redwood_views.sql 
+utl_encode_public.sql 
+utl_encode.plb 
+utl_http_public.sql 
+utl_http.plb 
+utl_file.plb 
+edb_ht_public.sql 
+edb_ht.plb 
+utl_tcp_public.sql 
+utl_tcp.plb 
+utl_smtp_public.sql 
+utl_smtp.plb 
+utl_mail_public.sql 
+utl_mail.plb 
+utl_url_public.sql 
+utl_url.plb 
+utl_raw_public.sql 
+utl_raw.plb 
+commoncriteria.sql 
+edb_gen_redwood.sql 
+waitstates.sql 
+installing extension edb_dblink_libpq ... ok
+installing extension edb_dblink_oci ... ok
+snap_tables.sql 
+snap_functions.sql 
+dblink_ora.sql 
+sys_stats.sql 
+ok
+finalizing initial databases ... ok
+syncing data to disk ... ok
+initdb: warning: enabling "trust" authentication for local connections
+initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
+Success. You can now start the database server using:
+    pg_ctl -D /var/lib/edb-as/15/dd12 -l logfile start
+
+```
+
+7. Start your database and navigate to your /data directory to view the postgresql.conf file to ensure that your `data_encryption_key_unwrap_command` that you set with your `export PGDATAUNWRAPCMD` is present under the Authentication section.
```

Comment:
This is not clear to me.

Authentication section?
Can we show the user an example?